actionlist/app/actions/types.js

```diff
@@ -1,5 +1,4 @@
-/* eslint-disable import/prefer-default-export */
-/* Needs to be done as actions are not dispatched without it.
-Will be removed when we have multiple actions */
-export const TOGGLE_ITEM = 'toggle_item';
-/* eslint-enable import/prefer-default-export */
+const TOGGLE_ITEM = 'toggle_item';
+const TOGGLE_DRAWER = 'toggle_drawer';
+export { TOGGLE_DRAWER, TOGGLE_ITEM };
```

Comment:
Is there a reason you're using this export and not exporting each line individually? It seems like more work to maintain it as you add new action types (and it's not consistent with the way the rest of the application exports things).